image/rhel/fetch-stackrox-data.sh

```diff
@@ -4,7 +4,11 @@
 
 set -euxo pipefail
 
+TARGET_DIR="${1:-/stackrox-data}"
+
 fetch_stackrox_data() {
+    local target_dir="${1}"
+
     mkdir -p /tmp/external-networks
```

Comment:
I think I may have reviewed this script but now the amount of times `/tmp/external-networks` is repeated worries me. If you don't mind improving things more, I suggest using `mktemp` and environment variable instead.

```suggestion
    local download_dir
    download_dir="$(mktemp -d --tmpdir external-networks.XXXXXXXXXX)"
```

`--tmpdir` and `external-networks.XXXXXXXXXX` pattern are optional cosmetics (in case they aren't available on all platforms).

Reply:
Done